processing/src/main/java/org/apache/druid/indexer/TaskStatus.java

```diff
@@ -37,7 +37,7 @@
  */
 public class TaskStatus
 {
-  public static final int MAX_ERROR_MSG_LENGTH = 100;
+  public static final int MAX_ERROR_MSG_TRUNCATION_LIMIT = 1024;
```

Comment:
I think the error msg will be shown in Druid console task status field, 1024 seems a bit long better check if it looks okay in the UI.

Reply:
Only the status is shown, the error message only seems to be only available in the per task view